homeassistant/components/scrape/strings.json

```diff
@@ -0,0 +1,73 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]",
+          "resource": "Resource",
+          "select": "Select",
+          "attribute": "Attribute",
+          "index": "Index",
+          "value_template": "Value Template",
+          "unit_of_measurement": "Unit of Measurement",
+          "device_class": "Device Class",
+          "state_class": "State Class",
+          "authentication": "Authentication",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "headers": "Headers"
+        },
+        "data_description": {
+          "resource": "The URL to the website that contains the value",
+          "select": "Defines the HTML tag to search for",
```

Comment:
This is confusing since it uses https://www.crummy.com/software/BeautifulSoup/bs4/doc/#css-selectors

Reply:
Right. Is was only copying from the integration page but I rewrote it slightly now and also put back the link on the integration page as it's pretty useful.